languages/thingtalk/dialogue_acts/action-confirm.js

```diff
@@ -68,7 +72,37 @@ function actionConfirmAcceptPhrase(ctx) {
     return clone.state;
 }
 
+function actionConfirmRejectPhrase(ctx) {
+    return new Ast.DialogueState(null, POLICY_NAME, 'cancel', null, []);
+}
+
+function actionConfirmChangeParam(ctx, answer) {
+    const questions = ctx.dialogueActParam || [];
```

Comment:
There won't be any questions at this template - it comes out of the agent asking for a confirmation, not asking for a parameter.